includes/admin/notes/note-functions.php

```diff
@@ -75,9 +75,14 @@ function edd_admin_get_note_html( $note_id = 0 ) {
 
 	// User
 	$user_id = $note->user_id;
-	$author  = ! empty( $user_id )
-		? get_userdata( $user_id )->display_name
-		: edd_get_bot_name();
+	$author  = edd_get_bot_name();
+	if ( ! empty( $user_id ) ) {
+		$author      = sprintf( '%s #%s', __( 'User ID', 'easy-digital-downloads' ), $user_id );
```

Comment:
This string could be given more context by not using a placeholder for `User ID`. @robincornett is there a specific reason to use a placeholder for that here?

Reply:
Sorry, with missing the text domain, I obviously was missing more than just that. Should be improved now.